index.js

```diff
@@ -245,8 +245,9 @@ module.exports = {
         if (piece.workflowGuid) {
           eventCopy.workflowGuid = self.apos.utils.generateId();
         }
-        eventCopy.parentId = piece._id;
+        eventCopy.parentId = piece.workflowGuid;
```

Comment:
What is the impact on existing data? Is a migration required here?

Reply:
I added a migration for that 👍